Assets/MixedRealityToolkit.Services/InputSimulation/Editor/InputSimulationService.cs

```diff
@@ -63,6 +63,7 @@ public override IMixedRealityController[] GetActiveControllers()
         /// <inheritdoc />
         public override void Initialize()
         {
+            ArticulatedHandPose.LoadGesturePoses();
```

Comment:
Curious, why do we need this, why did we not need this previously?

Reply:
Previously the poses were all hardcoded in SimulatedHandPose.cs as static fields with initializers. Right now we still have a fixed set of supported gestures in ArticulatedHandPose.GestureId, but we could perhaps make that configurable at some point (not sure how that might work, i'd rather not do that now).